packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -84,6 +85,11 @@
 #include "cuda_profiler_api.h"
 #endif
 
+// I am not 100% sure about the mpi 3 on cuda
```

Comment:
@kyungjoo-kim Are you asking whether MPI supports nonblocking collectives with CUDA buffers?  It should work but it might be slow.